config/settings/common.py

```diff
@@ -87,6 +87,8 @@
     'raven.contrib.django.raven_compat',
     'tos',
     'rest_framework',
+    
+    'django',
```

Comment:
Sorry, it is supposed to be 'djcelery', if django-celery is not needed, it
can be deleted.

On Thu, Mar 3, 2016 at 4:44 PM, Nicholas Serra <notifications@github.com>
wrote:

> In config/settings/common.py
> <https://github.com/SEED-platform/seed/pull/817#discussion_r54953432>:
>
> > @@ -87,6 +87,8 @@
> >      'raven.contrib.django.raven_compat',
> >      'tos',
> >      'rest_framework',
> > +
> > +    'django',
>
> Why was this added? Looks incorrect.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/817/files#r54953432>.
>